### DISCUSSION TITLE
APIs functionality

### DISCUSSION THREAD

**Discussion by amitjattan** [Azure/ipam#180] [General]
@DCMattyG Wondering if those updates are in place yet

1. Restrict Reservations APIs to Admin users/SPNs
2. Subnet APIs for regular users

**Comment by DCMattyG** [comment]
Hi @amitjattan, long time no talk buddy!

1. I've just completed a ton of major updates and security fixes. Now that this work is complete, I'll shift my focus to this and check-in with you as I make progress.
2. Can you remind me of some more specifics on this item please? A lot has been going on and I just want to make sure I address this correctly.

Thanks so much!
